client/my-sites/checkout/checkout/domain-refund-policy.jsx

```diff
@@ -111,8 +111,8 @@ class DomainRefundPolicy extends React.Component {
 
 		return (
 			<>
-				{ policies.map( ( policy ) => (
-					<div className="checkout__domain-refund-policy">
+				{ policies.map( ( policy, index ) => (
```

Comment:
It kind of defeats the purpose of a `key` to use the array index ([one article on the topic](https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318)). It would be better to use something that is unique about the array element. In this case, I think that `policy` might be a good choice.

Reply:
TIL